rpcs3/Emu/system_config.h

```diff
@@ -43,6 +43,7 @@ struct cfg_root : cfg::node
 		cfg::_bool spu_accurate_dma{ this, "Accurate SPU DMA", false };
 		cfg::_bool accurate_cache_line_stores{ this, "Accurate Cache Line Stores", false };
 		cfg::_bool rsx_accurate_res_access{this, "Accurate RSX reservation access", false, true};
+		cfg::_bool rsx_accurate_fifo_access{this, "Accurate RSX FIFO access", true };
```

Comment:
This needs to be in the UI. The default is to keep current behaviour unless it is proven to be a purely enhancing feature, which I'm still evaluating. I doubt anyone compared or even tested if inaccurate mode even works. Nobody edits configs these days.